src/libmongoc/tests/bsonutil/bson-val.h

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
Should we make the return values const for the getters below that return pointers to the internals of structs? I can't think of a good reason to change the returned values that way.

Reply:
Sam also brought this up in https://github.com/mongodb/mongo-c-driver/pull/729#discussion_r564154605
But, I relent. It does convey intent in API to make things const correctness – even in test code. Updating.